api/lib/opentelemetry/baggage/propagation/text_map_extractor.rb

```diff
@@ -7,41 +7,41 @@
 require 'cgi'
 
 module OpenTelemetry
-  module CorrelationContext
+  module Baggage
     module Propagation
-      # Extracts correlations from carriers in the W3C Correlation Context format
+      # Extracts baggage from carriers in the W3C Baggage format
       class TextMapExtractor
         include Context::Propagation::DefaultGetter
 
         # Returns a new TextMapExtractor that extracts context using the specified
         # header key
         #
-        # @param [String] correlation_context_key The correlation context header
+        # @param [String] baggage_key The baggage header
         #   key used in the carrier
         # @return [TextMapExtractor]
-        def initialize(correlation_context_key: 'otcorrelations')
-          @correlation_context_key = correlation_context_key
+        def initialize(baggage_key: 'Baggage')
```

Comment:
It's worth mentioning that this change is still under discussion: https://github.com/open-telemetry/opentelemetry-specification/issues/879. It will probably go through, although, I believe we should go with all lowercase `baggage`, as the canonical key. For http it doesn't matter, but for case-sensitive scenarios I believe we want to standardize on using all lower case.

Reply:
I speculated that the change would go through. If it's rejected, I can change back. I canonicalized the key.